media/js/plugincheck/plugincheck.js

```diff
@@ -10,9 +10,10 @@
 
     if (client.isFirefoxDesktop) {
         client.getFirefoxDetails(function(details) {
-            if (details.isUpToDate && details.channel === 'release') {
+            var isUpToDate = client._isFirefoxUpToDate(false, details.isESR, details.version);
```

Comment:
Only thought here is to add a comment explaining when `isUpToDate` is true and when it's false (as the method call/signature doesn't make it obvious).